src/gql/queries/get-mainline-commits.graphql

```diff
@@ -1,5 +1,6 @@
 query MainlineCommits(
   $mainlineCommitsOptions: MainlineCommitsOptions!
+  $buildVariantOptionsForTask: BuildVariantOptions!
   $buildVariantOptions: BuildVariantOptions!
```

Comment:
Why are these different? Is this so we can show the overall task stats but only show build variant cards that match the filter criteria for the default view?

Reply:
They are different because for the default`taskStatusCounts` (the thing that renders active commit charts) aren't filtered by any status but by default build variant cards are filtered by all the failure statuses.